src/csm_testing/tests/sat_functional/hwinv/test_hwinv.py

```diff
@@ -0,0 +1,630 @@
+    'Drive Count', 'Total Drive Capacity (GiB)', 'BIOS Version', 'Memory Model'
+]
+HWINV_CHASSIS_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU'
+]
+HWINV_HSN_BOARDS_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU'
+]
+HWINV_COMPUTE_MODULES_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU'
+]
+HWINV_ROUTER_MODULES_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU'
+]
+HWINV_NODE_ENCLOSURES_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'SKU', 'Serial Number'
+]
+HWINV_NODE__ENCLOSURES_POWER_SUPPLIES_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU'
+]
+HWINV_PROCS_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU',
+    'Total Cores', 'Total Threads', 'Max Speed (MHz)'
+]
+HWINV_NODE_ACCELS_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU',
+    'Location Name'
+]
+HWINV_NODE_ACCEL_RISERS_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU',
+    'Board Serial Number', 'Producer', 'Engineering Change Level'
+]
+HWINV_NODE_HSN_NICS_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU'
+]
+HWINV_MEMS_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU',
+    'Memory Type', 'Device Type', 'Capacity (MiB)', 'Operating Speed (MHz)'
+]
+HWINV_DRIVES_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU',
+    'Media Type', 'Capacity (GiB)', 'Percent Life Left'
+]
+HWINV_CMM_RECTIFIERS_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU',
+    'Power Supply Type', 'Firmware Version', 'Power Input Watts', 'Power Output Watts'
+]
+HWINV_NODE_BMCS_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU',
+    'Manager Type', 'Firmware Version'
+]
+HWINV_ROUTER_BMCS_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU',
+    'Manager Type', 'Firmware Version'
+]
+HWINV_MGMT_SWITCHES_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU',
+    'Chassis Type'
+]
+HWINV_CABINET_PDUS_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU',
+    'Equipment Type', 'Firmware Version'
+]
+HWINV_CABINET_PDU_POWER_CONNECTORS_HEADER = [
+    'xname', 'FRUID', 'Manufacturer', 'Model', 'Part Number', 'Serial Number', 'SKU',
+    'Nominal Voltage', 'Outlet Type', 'Phase Wiring Type', 'Power Enabled',
+    'Rated Current Amps', 'Voltage Type'
+]
+
+def adjust_expected_header(header: List[str], sat_output: str) -> List[str]:
```

Comment:
PEP8 wants an additional blank line

```suggestion

def adjust_expected_header(header: List[str], sat_output: str) -> List[str]:
```